soc/arm/silabs_exx32/efm32pg1b/Kconfig.defconfig.series

```diff
@@ -0,0 +1,20 @@
+# EFM32PG1B series configuration options
+
+# Copyright (c) 2020 Rafael Dias Menezes <rdmeneze@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_EFM32PG1B
+
+config SOC_SERIES
+	default "efm32pg1b"
+
+config SOC_PART_NUMBER
+	default "EFM32PG1B200F256GM48" if SOC_PART_NUMBER_EFM32PG1B200F256GM48
+
+config NUM_IRQS
+	# must be >= the highest interrupt number used
+	default 30
```

Comment:
Interrupt FPUEH is IRQ number 33, so this should be 33 or 34 (i am not sure which one). For efm32pg12 is is one above FPUEH. So 34 would be the equivalent value here.